databricks/koalas/missing/series.py

```diff
@@ -55,7 +55,6 @@ class _MissingPandasLikeSeries(object):
     compress = unsupported_function('compress')
     convert_objects = unsupported_function('convert_objects')
     copy = unsupported_function('copy')
-    corr = unsupported_function('corr')
```

Comment:
how is this change adding corr to Series? Do all the methods that are added to Frame automatically get added to Series?

Reply:
Yes, Frame is the base class for both `Series` and `DataFrame`. In practice, though, the implementations are usually different enough that functions are added in each of `frame.py` and `series.py`.